react/components/QuickOrderAutocomplete.tsx

```diff
@@ -4,9 +4,15 @@ import { AutocompleteInput } from 'vtex.styleguide'
 import PropTypes from 'prop-types'
 import { WrappedComponentProps, injectIntl, defineMessages } from 'react-intl'
 import { useApolloClient } from 'react-apollo'
+import { useCssHandles } from 'vtex.css-handles'
 
 import autocomplete from '../queries/autocomplete.gql'
 
+const CSS_HANDLES = [
+  'vtex-button',
```

Comment:
Please use a naming convention similar to the other CSS handles in the app. It should be clear from the handle name which button this is referencing, i.e. `customOptionButton` or similar.

Reply:
We need to add a class which starts with vtex- to perform customization in our application. Hence we cannot use any other names apart from this.

Reply:
@naveenpr1993 That's true for global CSS styles, but the preferred method for styling app components is to create a CSS file in your store-theme using the name of the app (i.e. `vtex.quickorder.css` in this case) and then using a CSS selector targeting the CSS handle (i.e. `.customOptionButton { ...`). See documentation here: https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-using-css-handles-for-store-customization

Reply:
Alternatively, you can target the suggested CSS handle in your global CSS. The final rendered class name will work out to be  `.vtex-quickorder-3x-customOptionButton`

Reply:
Yes you can added them here in this PR

Reply:
Raised a new PR https://github.com/vtex-apps/quickorder/pull/109 with all the changes